fastlane/Fastfile

```diff
@@ -940,6 +940,59 @@ REPOSITORY_NAME="WordPress-Android"
     )
   end
 
+  #####################################################################################
+  # create_gh_release
+  # -----------------------------------------------------------------------------------
+  # This lane creates a GitHub release for the current version
+  #   - Attaching the existing .aab files in `build/` as the release assets
+  #   - Using the WP & JP release_notes.txt as description
+  #   - If prerelease:<true|false> is not provided, the pre-release status will be guessed from the version name
```

Comment:
```suggestion
  #   - If `prerelease:<true|false>` is not provided, the pre-release status will be guessed from the version name
```

Reply:
Fixed in f2a3e704a6eb47da9d58ff527cca6b89e01c0cb4; also changed the wording from "guessed" to "inferred" (as that was what I originally meant, but couldn't remember the word yesterday 😄)

Reply:
> This is very neat. Blown away by the speed at which CI built. I went on the tab to check on the progress and it had already finished! 🤯
 
Yep, definitively! Having this project done (paaHJt-2s8-p2 + paaHJt-2s9-p2) will change my life as Release Manager!! I'm so happy for this change, no more spending the whole morning waiting for CI (and having to wait for WP CI to finish before I need to trigger JP CI) and no more having to do half of the tasks by hand anymore 🎉  #relief

> Not sure what your plans for merging are. In case you haven't thought of it already, I'd suggest rewriting the history to `drop` the last two commits.

Yeah I did it that way because last time I used a debug commit like that, [the PR was accidentally merged before we pushed back the revert commit](https://github.com/wordpress-mobile/WordPress-Android/pull/15273#issuecomment-915084562), so I figured it was better to include the debug commit and its revert this time, and ask you to point before the revert commit. 

That being said, I think I'll keep those 2 commits in the merge, because rewriting history to drop them would make the PR description invalid if I did, and it can still make sense to keep track on how the PR was tested. Not a big deal once the PR is merged you'd say, and I can also see the value in keeping the git history cleaner, but I still like to keep consistency in the PR comments and desc even post-merge, in case we need to come back to this PR later one day and need to make sense of it 🙃